src/peft/peft_model.py

```diff
@@ -1065,16 +1070,17 @@ def forward(
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
         if not isinstance(peft_config, PromptLearningConfig):
-            return self.base_model(
-                input_ids=input_ids,
-                attention_mask=attention_mask,
-                inputs_embeds=inputs_embeds,
-                labels=labels,
-                output_attentions=output_attentions,
-                output_hidden_states=output_hidden_states,
-                return_dict=return_dict,
-                **kwargs,
-            )
+            kwargs.update({
+                "input_ids": input_ids,
+                "attention_mask": attention_mask,
+                "inputs_embeds": inputs_embeds,
+                "output_attentions": output_attentions,
+                "output_hidden_states": output_hidden_states,
+                "return_dict": return_dict,
+            })
+            if labels is not None:
+                kwargs["labels"] = labels
```

Comment:
Hello, I don't see the need for this, if user doesn't pass labels, it will be `None` which should work fine

Reply:
That doesn't work if the underlying model doesn't have the `labels` parameter in its forward pass.  A model such as `roberta-large` or `vinai/phobert-large` has that problem.  I go into more details in the linked issue:  https://github.com/huggingface/peft/issues/348

Actually, I think `TOKEN_CLS` isn't exactly the right way to make use of those models.  Currently I don't believe there's any way to use peft to finetune `roberta-large` without a classifier layer, though.  Perhaps a new version such as `ENCODER` would be more appropriate.  Although in the end that would share quite a lot of code with `TOKEN_CLS`